<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Borne</class>
 <widget class="QWidget" name="Borne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Borne</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_40">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>BORNES</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="TableBorne">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Nom</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Prix</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Num√©ro</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="AddButton">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DelButton">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nom: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNom"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Prix: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPrix">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Numero</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNum"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editType"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
